deployment/terraform/cdn.tf

```diff
@@ -8,7 +8,6 @@ resource "aws_cloudfront_distribution" "tilegarden" {
     }
 
     domain_name = "${var.tilegarden_api_gateway_domain_name}"
-    origin_path = "/latest"
```

Comment:
Now that we're routing `/latest` requests directly to the cache, we no longer need to set up this path in the origin config.